src/components/Dropdowns/Dropdowns.scss

```
@@ -0,0 +1,91 @@
+@import "../../styles/vars.scss";
+
+.dropdowns {
+  &__content {
+    position: relative;
+    display: flex;
+    gap: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__text {
+    margin: 0 0 14px;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 15px;
+    color: $secondaryColor;
+  }
+
+  &__button {
+    box-sizing: border-box;
+
+    position: relative;
+    height: 40px;
+    width: 176px;
+    margin: 0 0 4px;
+    padding-left: 12px;
+
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 21px;
+    color: $primaryColor;
+    text-align: left;
+
+    border: 1px solid $iconsColor;
+    background-color: $whiteColor;
+
+    &--small {
+      width: 128px;
+    }
+
+    &:hover {
+      border: 1px solid #89939a;
```

search in file `#` & replace all colors with variables
```suggestion
      border: 1px solid $secondaryColor;
```